september/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -85,6 +85,10 @@
 		<hr style="margin-bottom: 25px;">
 		<p class="text-muted pull-left">
 		
+		<c:set var="PR" value="${pullrequest}" />
+            Version: ${geco.september.version}<c:if test="${not empty PR}">, Pull Request: ${pullrequest}</c:if> 
```

Comment:
let's use the buildnumber plugin, so that buildnumbers are generated unless it's a stable version. the other projects should have examples on how to do that.